README.md

```diff
@@ -1,3 +1,5 @@
+**[DEPRECATED]** _This repository is no longer used, as in Gutenberg we now use the original version of this package._
```

Comment:
Hmm, I'm thinking, while we do use the NPM package directly from npmjs.org, there's the "magic sauce" of using a S3-kept, separately built Android artifact, right? Perhaps the deprecation notice can point to a resource describing that (could be a link to a readme.md page for example). WDYT?

Reply:
> Hmm, I'm thinking, while we do use the NPM package directly from npmjs.org, there's the "magic sauce" of using a S3-kept, separately built Android artifact, right?

Exactly, the NPM package is fetched from the NPM registry and the Android artifact from our Maven repository in S3.

> Perhaps the deprecation notice can point to a resource describing that (could be a link to a readme.md page for example). WDYT?

Good idea, what do you think about using [this README](https://github.com/wordpress-mobile/react-native-libraries-publisher/blob/trunk/README.md) which describes the process of publishing artifacts? Maybe it doesn't describe accurately the "magic sauce", but we could include extra information regarding this.

Reply:
I think that README would work, here's one way to point to it perhaps:

```suggestion
**[DEPRECATED]** _This repository is no longer used, as in Gutenberg we now use the original version of this package, plus fetching [a prebuilt Android artifact from a maven repo on S3](https://github.com/wordpress-mobile/react-native-libraries-publisher/blob/trunk/README.md)._
```

Reply:
Great, the change you suggested looks good, I'll add it 🙇 .